proposals/2021-04-advancing-ethermint/README.md

```diff
@@ -0,0 +1,117 @@
+
+Ethermint is a way of vertically and horizontally scaling the projects that deploy on Ethereum, by allowing:
+
+- Access to greater liquidity through IBC
+- Faster transaction times through Tendermint BFT's instant finality
+- Less strain on the Ethereum chain to process transactions (i.e. through gravity bridge)
+- Seamless portability and composability with Ethereum Virtual Machine support
+
+The commitments stated in this proposal will ensure a successful launch for the Ethermint chain together with its existing contributors (ChainSafe, OKEx, Iris, Injective, etc). Launching collaboratively with Ethermint stakeholders will result in a robust and independent community within Cosmos that will enable greater developer adoption of Cosmos technology.
+
+The Ethermint stakeholders are partnering to execute on the long-awaited Ethermint Chain by joining forces to develop and deploy a new Cosmos EVM chain that will be used by thousands of crypto users at launch.
+
+## Governance Votes
+
+The following items summarize the voting options and what it means for this proposal. All addresses that vote on the proposal might be eligible for a future airdrop.
+
+- **YES**: You approve the proposal statements and distribute the amount of 100,000 ATOMs to the multisig address. The treasury will allocate the funds to the Tharsis team, leading Ethermint's core development efforts and supporting the chain's go-to-market strategy. It will also lead to core technology maintenance and continuous discussion to ensure the project's longevity.
+- **NO**: The NO vote is a request for improvements or adjustments. You agree that this proposal's motivation is valuable and that the team should create a follow-up proposal once the amendments are included.
+- **NO (VETO)**: You veto the entire motivation for the proposal and expect the ICF and current maintainers to make the determination and continue the stewardship of the project. The proposers will not create a follow-up proposal.
+- **ABSTAIN**: You are impartial to the outcome of the proposal.
+
+## Multisig and release of funds
+
+Upon the approval of the proposal, the treasury will distribute the funds to a ⅔ multi-signature account managed by the following individuals/partners:
+
+- Federico Kunze Küllmer - Tharsis (proposer)
+- Zaki Manian - Iqlusion
+- Marko Baricevic - Interchain GmbH
+
+<!-- TODO: add -->
+The account address is: cosmos1XXX
+
+If the proposal passes, the team will immediately receive 40% of the funds to expand its engineering team and other business development efforts to support GTM for the chain. The remaining 60% will be released in an equal proportion to the number of milestones upon the completion of each milestone. For any reason, if the proposer has not completed the next milestone within a year of the last payment, the remaining funds held in the multisig account will be returned to the community pool.
```

Comment:
Perhaps adding in "We give permission for governance to perform an irregular state change to this account, in the event that any of the above process was not followed"

Reply:
Marko and I will just just send the funds back to the community pool. That is my expectation

Reply:
^Agreed, I think the multi-sig is pretty sufficient, and basically the funds that are spent shouldn't come back. Has to be some risk expected. Otherwise you'd be minting more money (bad) or you'd be reverting transaction (also bad).